InnovativeCars/force-app/main/default/classes/RenovationTeamRequestSender.cls

```diff
@@ -0,0 +1,44 @@
+/**
+* @author Agnieszka Ząbkowicz
+* @date 06.2021
+*
+* @description class for sending request to renovation team
+* when a showroom is being closed
+*/
+public class RenovationTeamRequestSender implements Queueable, Database.AllowsCallouts{
+    private List<Showroom__c> closedShowrooms;
+
+    public RenovationTeamRequestSender(List<Showroom__c> showrooms){
+        this.closedShowrooms = showrooms;
+    }
+
+    public void execute(QueueableContext context){
+        List<Case> newCases = new List<Case>();
+        for(Showroom__c showroom : closedShowrooms){
+            newCases.add(new Case(
+                Subject = Constants.CASE_SUBJECT + showroom.Name, 
+                Showroom__c = showroom.Id, 
+                Origin = Constants.CASE_ORIGIN
+                ));
+        }
+
+        Http server = new Http();
+        HttpRequest req = new HttpRequest();
+        req.setEndpoint(Constants.RENOVATION_ENDPOINT);
```

Comment:
consider having endpoint in named credentials / custom metadata / custom setting